doc/source/admin/galaxy_options.rst

```diff
@@ -3823,6 +3823,43 @@
 :Type: str
 
 
+~~~~~~~~~~~~~~~~~~~~
+``ga4gh_service_id``
+~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    Service ID for GA4GH services (exposed via the service-info
+    endpoint for the Galaxy DRS API). If unset, one will be generated
+    using the URL the target API requests are made against.
```

Comment:
This should be `usegalaxy.org` for example I assume? I guess this is important to set, especially if you have subdomains.

Reply:
Can we give here and maybe below a best-practice recommendation?

Reply:
I'm sure it isn't important to set at all 😆 but I've updated the documentation with links and my best guess at a recommendation for usegalaxy.org. Thanks for the attention to detail!